templates/test/ci/prow-azl3/patches/cloud-provider-azure-ci-cacertdir.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: addons.cluster.x-k8s.io/v1alpha1
+kind: HelmChartProxy
+metadata:
+  name: cloud-provider-azure-chart-ci
+spec:
+  valuesTemplate: |
+    infra:
+      clusterName: {{ .Cluster.metadata.name }}
+    cloudControllerManager:
+      caCertDir: "/etc/pki/tls/certs"
```

Comment:
Sorry I gave you bad advice here! (Although I swear `/etc/pki/tls` made Azure cloud controller happy when I previously tested--maybe Mariner differs from Azure Linux 3 in this.)